entity/index.js

```diff
@@ -1447,12 +1451,31 @@ module.exports = EntityGenerator.extend({
             } else {
                 this.pkType = 'Long';
             }
+
+            var entityNameSpinalCased = _s.trim(_s.dasherize(this.name), '-');
+            var entityNamePluralizedAndSpinalCased = _s.trim(_s.dasherize(pluralize(this.name)), '-');
+
+            this.entityApiUrl = entityNamePluralizedAndSpinalCased;
```

Comment:
What do you think of the entity prefix for for all in-memory variables. Should we remove them? The problem with that is that they could overwrite app properties. 
An option would be to put them in an object defined as 'e'  or 'entity' though that will be more verbose. 
I'm a bit unsure about this. Any suggestions?